### PR TITLE
New package: dockstar-base-1

### PR DESCRIPTION
**Pull request by thypon**

### CODE REVIEW

File: srcpkgs/dockstar-base/INSTALL [hunk 1]
```diff
@@ -0,0 +1,8 @@
+case "$ACTION" in
+post)
```
Comment (beefcurtains) [inline srcpkgs/dockstar-base/INSTALL]: Please also check if ``$UPDATE`` is ``no`` (or not ``yes``)

Comment (thypon) [inline srcpkgs/dockstar-base/INSTALL]: It's done following the odroid base. All the things i run inside are idempotent commands, then checking for UPDATE it's not necessary for now.

Comment (beefcurtains) [inline srcpkgs/dockstar-base/INSTALL]: It's not that they might fail, it's that these services are enabled whenever the package is updated. But, perhaps this should be fixed in a separate bug/pr. You have my :ok:

File: srcpkgs/dockstar-base/template [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Template file for 'dockstar-base'
+pkgname=dockstar-base
+version=1
+revision=1
+build_style=meta
+homepage="http://www.voidlinux.eu"
+short_desc="Void Linux Dockstar platform package"
+maintainer="Andrea Brancaleoni <miwaxe@gmail.com>"
+license="Public Domain"
+noarch=yes
```
Comment (xtraeme) [inline srcpkgs/dockstar-base/template]: Please drop `noarch`. This package is only meant to be available for the target platform, and not to be shared by all architectures in the same repository.

**Pull request closed without merging**